src/app/services/getDocuments.js

```diff
@@ -0,0 +1,35 @@
+import http from "@/app/http"
+
+export async function getDocuments() {
```

Comment:
perhaps a slightly different name to make it less confusing? `getAllDocuments`, `queryDocuments`? dunno